ext/nnstreamer/tensor_converter/tensor_converter_flatbuf.cc

```diff
@@ -0,0 +1,130 @@
+/**
+ * GStreamer / NNStreamer tensor_converter subplugin, "Flatbuffer"
+ * Copyright (C) 2020 Gichan Jang <gichan2.jang@samsung.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
```

Comment:
For new files, please use SPDX format instead of the full boilerplate.

Reply:
Updated.